project/dimensions/model_years.csv

```diff
@@ -0,0 +1,11 @@
+id,name
+2020,2020
+2022,2022
+2024,2024
+2026,2026
+2028,2028
+2030,2030
+2035,2035
+2040,2040
+2045,2045
+2050,2050
```

Comment:
These years generally align with Cambium. (Although Cambium only starts at 2024.)

Reply:
As we've discussed, for buildings we plan to produce results for 2035 and 2050. At one point we discussed needing to produce results for a base year (e.g., 2020 or 2025) — is that not your expectation @elainethale ?

Reply:
@asatremeloy, that is my expectation. We'll just need to specify factors that interpolate your results to these years so we can provide data to off-takers.